src/pages/Product/index.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ProductComponent from '../../components/Product/Product';
+
+const index = () => (
+  <ProductComponent />
+);
+
+export default index;
```

Comment:
what is the use of this component, as it does no thing rather than representing another component